ffmpeg/VITABUILD

```diff
@@ -0,0 +1,55 @@
+pkgname=ffmpeg
+pkgver=3.3.9
+pkgrel=1
+url="http://ffmpeg.org/"
+source=("http://ffmpeg.org/releases/ffmpeg-$pkgver.tar.xz" "ffmpeg.patch")
+sha256sums=('ae34f14fffa65a1a59b256737ca9af7bf4e296b7c4320d42512350126ce06c84' 'c4058a723957547c5450e4c2694eb0b9393d983146079382f311be77f9d87f75')
+
+prepare() {
+  cd $pkgname-$pkgver
+  patch -Np1 -i "${srcdir}/ffmpeg.patch"
+}
+
+build() {
+  cd $pkgname-$pkgver
+  ./configure --prefix=$prefix \
+		--enable-cross-compile \
+		--cross-prefix=$VITASDK/bin/arm-vita-eabi- \
+		--disable-shared \
+		--disable-runtime-cpudetect \
+		--disable-armv5te \
+		--disable-programs \
+		--disable-doc \
+		--disable-network \
+		--enable-gpl \
```

Comment:
an application developers have to get a chance to decide their licence. but `--enable-gpl` option could make the problem to it.

Reply:
Agree. From their documentation

Compile FFmpeg without "--enable-gpl" and without "--enable-nonfree".

Reply:
I've already pushed a commit that removes it. Without those, ffmpeg will be compiled with LGPL license.